### DISCUSSION TITLE
Custom Wallpaper requesting.

### DISCUSSION THREAD

**Discussion by SmokeyPlayz** [zen-browser/desktop#2174] [Ideas]
This is a small feature that can be put out to allow users to upload a custom wallpaper to they're "new tab" Page quite unfortunate that this browser  haven't yet added The custom wallpaper feature and also one more thing, can their be more performance settings or something to boost the browser Speed like the browser thorium i did use that one before this one but, For me i had some problems so when i heard about zen i just straight on installed it on my linux laptop then my windows 11 pc and as the sight is improving more perhaps there can be like mods to apply to help maybe lower the resources the browser use for the best user-friendly performance stability.

**Comment by daviddanko** [comment]
I would also really like to have a custom wallpaper on my homepage. Maybe even adding a bundle of photos which we could change manually or set to change automatically in intervals - this would be the top of the cake.
